querybook/webapp/redux/dataTableSearch/action.ts

```diff
@@ -90,9 +90,17 @@ function searchDataTable(): ThunkResult<Promise<ITableSearchResult[]>> {
             if (state.searchRequest) {
                 state.searchRequest.cancel();
             }
-            const searchRequest = SearchTableResource.searchConcise(
-                mapStateToSearch(state)
-            );
+            const tableSort = state.sortTablesBy;
+            const search = { ...mapStateToSearch(state) };
+            if (tableSort.key === 'name') {
+                const tableSortAsc = tableSort.asc ? 'asc' : 'desc';
+                search['sort_key'] = ['schema', 'name'];
+                search['sort_order'] = [tableSortAsc, tableSortAsc];
+            } else if (tableSort.key === 'importance_score') {
+                search['sort_key'] = '_score';
+                search['sort_order'] = tableSort.asc ? 'asc' : 'desc';
```

Comment:
What if you wanted to find unimportant tables to clean them up? 😄 

I think this would have to be paired with hiding the direction toggle on the UI when importance is selected.

Reply:
Yeah I agree 😅 
It's a bit misleading to have the direction toggle but not being able to use it, I can hide it when sorting by importance_score